verifier/src/report_body.rs

```diff
@@ -811,10 +856,14 @@ mod test {
 
         let mr_signer_verifier = MrSignerVerifier::new(mr_signer, product_id, isv_svn);
 
-        assert_eq!(
-            mr_signer_verifier.verify(&report_body).is_ok().unwrap_u8(),
-            1
-        );
+        let verification = mr_signer_verifier.verify(&report_body);
+        assert_eq!(verification.is_ok().unwrap_u8(), 1);
+        let expected = r#"
+            - [x] MrSigner:
+              - [x] The MRSIGNER key hash was MrSigner(sgx_measurement_t { m: [48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79] })
```

Comment:
See, https://github.com/mobilecoinfoundation/sgx/issues/290 for making the display nicer, may want to prioritize